test/WebJobs.Script.Tests/app.config

```diff
@@ -1,39 +1,53 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <connectionStrings>
+    <add name="AzureWebJobsStorage" connectionString="DefaultEndpointsProtocol=https;AccountName=brettsamwebjobs;AccountKey=AW+LPMa3PaZoJzr8cOTJ5/UKMJkBd+4yLuLrfWT/F+0/mspGqnlSvAUjjuMhvyeP9fB6VM68xeVNCxOUjLYv8A==;BlobEndpoint=https://brettsamwebjobs.blob.core.windows.net/;TableEndpoint=https://brettsamwebjobs.table.core.windows.net/;QueueEndpoint=https://brettsamwebjobs.queue.core.windows.net/;FileEndpoint=https://brettsamwebjobs.file.core.windows.net/" />
+  </connectionStrings>
+  <appSettings>
+    <add key="AzureWebJobsEventHubReceiver" value="" />
+    <add key="AzureWebJobsEventHubSender" value="" />
+    <add key="AzureWebJobsEventHubPath" value="" />
+    <add key="AzureWebJobsMobileAppUri" value="https://brettsameasytables.azurewebsites.net" />

```

Comment:
Done.

---
In reply to: [56192597](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56192597) [](ancestors = 56192597)